modules/ROOT/pages/connecting.adoc

```diff
@@ -1,15 +1,27 @@
 = Connecting to Your ownCloud
 :toc: right
-
 :app-name: ownCloud Android App
 :create-self-signed-ssl-cert: https://www.digitalocean.com/community/tutorials/how-to-create-a-self-signed-ssl-certificate-for-apache-in-ubuntu-16-04
 :enable-ssl-url: http://info.ssl.com/article.aspx?id=10241
 :oauth2-app-url: https://marketplace.owncloud.com/apps/oauth2
+:document-provider-url: https://developer.android.com/guide/topics/providers/document-provider
 
 == Introduction
 
 This section describes how to connect the {app-name} to your ownCloud.
 
+== Physical Location of User Files
+
+Compared to earlier versions of this app (< 2.19), the location for files created or managed by this app is the so called *Scoped Storage*. Scoped Storage is, compared to Shared Storage, protected and part of the app. It is located on the devices memory and limited by its capacity. Files are now safe and private and can only be accessed by other apps that use the {document-provider-url}[Documents Provider]. Internal tests have successfully passed using more than 50GB of Scoped Storage without any issues.
```

Comment:
```suggestion
Compared to earlier versions of this app (< 2.19), the location for files created or managed by this app is called *Scoped Storage*. Scoped Storage is, in contrast to Shared Storage, protected and part of the app. It is saved in the device's memory and limited by its capacity. Files are now safe and private and can only be accessed by other apps that use the {document-provider-url}[Documents Provider]. Internal tests have successfully passed using more than 50GB of Scoped Storage without any issues.
```

Reply:
committing, will adopt the memory -> "storage" part later